### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by raproid**
Updated README.md:

- Added main heading.
- Added Introduction.
- Isolated Go installation into the "Prerequisites section" and expanded it.
- Added a little more info into the "Deploy locally" section.
- Minor punctuational and syntactical fixes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,45 +1,108 @@
...
+* Transfer XNS to Insolar MainNet users
+
+* Receive XNS from Insolar MainNet users
+
+
+## Quick start
+
+You can test Insolar Mainnet locally:
+
+1. Install everything from the **Prerequisites** section.
```
Comment (megge-dream) [inline README.md:25]: Is Prerequisites correct word here?

File: README.md [hunk 2]
```diff
@@ -1,45 +1,108 @@
...
+
+Install Golang programming tools **v1.12**:
+
+#### Fresh Go installation
+
+* Linux and Windows: install the tools from [goland.org](https://golang.org/doc/install#install). Set [`$GOPATH` environment variable](https://github.com/golang/go/wiki/SettingGOPATH).
+
+* macOS: use Homebrew to install the tools: `brew install go@1.12`. Set [`$GOPATH` environment variable](https://github.com/golang/go/wiki/SettingGOPATH).
+
+#### Multiple versions of Go on Windows
```
Comment (megge-dream) [inline README.md:40]: Not sure about this

File: README.md [hunk 3]
```diff
@@ -1,45 +1,108 @@
...
+ 
+1. In the directory where you downloaded the Mainnet package to, run the launcher:
 
    ```
    insolar-scripts/insolard/launchnet.sh -g
    ```
 
-   It generates bootstrap data, starts a pulse watcher, and launches a number of nodes. In local setup, the "nodes" are simply services listening on different ports.
-   The default number of nodes is 5, you can uncomment more in `scripts/insolard/bootstrap_template.yaml`.
+   The launcher generates te necessary bootstrap data, starts a pulse watcher, and launches a number of nodes. <br>
```
Comment (megge-dream) [inline README.md:98]: The launcher generates **the** necessary

File: README.md [hunk 4]
```diff
@@ -49,23 +112,25 @@ To test Mainnet locally, install it and deploy as described below.
...
+Thank you for your intention to contribute to the Insolar Mainnet project. As a company developing open-source code, we highly appreciate external contributions to our project.
 
-Thank you for your intention to contribute to the Mainnet project. As a company developing open-source code, we highly appreciate external contributions to our project.
+## Contacts
 
-# Contacts
+If you have any additional questions, join our [developers chat on Telegram](https://t.me/InsolarTech).
 
-If you have any additional questions, join our [developers chat](https://t.me/InsolarTech).
+<!-- The icons don't show!
```
Comment (megge-dream) [inline README.md:133]: Why? I can see them in master branch

File: README.md [hunk 5]
```diff
@@ -1,45 +1,108 @@
...
+Insolar MainNet is the application that implements smart contracts logic for Insolar MainNet. 
 
-To test Mainnet locally, install it and deploy as described below.
+This application works ontop of [Insolar platform](https://github.com/insolar/insolar) and allows you to:
 
-## Install
+* Create Insolar wallet
 
-1. Install the latest 1.12 version of the [Golang programming tools](https://golang.org/doc/install#install). Make sure the `$GOPATH` environment variable is set.
+* Migrate INS onto Insolar exchanging INS to XNS 1:1 where INS is Ethereum ERC-20 token and XNS is Insolar native token.
```
[the remark this answers is not in the document]
Comment (raproid) [reply README.md:12]: what is the ratio?

File: README.md (hunk 1 above)
Comment (raproid) [reply README.md:25]: Yes. Everything beyond the actual installation is prerequisites. Go is something implied here: you need to have a correct and set version of Go.

File: README.md (hunk 2 above)
Comment (raproid) [reply README.md:40]: Not sure about macOS or Windows?

I tested installation of go@1.12 on macOS.

Windows - didn't test it.

File: README.md (hunk 3 above)
Comment (raproid) [reply README.md:98]: tnx

File: README.md (hunk 4 above)
Comment (raproid) [reply README.md:133]: Really?

Comment (raproid) [reply README.md:133]: Oh, snap, it's my adblocker. tnx

File: README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (megge-dream) [reply README.md:12]: 1:10

File: README.md (hunk 2 above)
Comment (megge-dream) [reply README.md:40]: Windows

Comment (LinnikD) [reply README.md:40]: remove windows pls

File: README.md [hunk 6]
```diff
@@ -1,45 +1,107 @@
...
+1. Install everything from the **Prerequisites** section.
+2. Install this application.
+3. Deploy this application locally.
+4. Test this application locally.
+
+### Prerequisites
+
+Install Golang programming tools **v1.12**:
+
+#### Fresh Go installation
```
Comment (LinnikD) [inline README.md:34]: It is not common for golang projects on github to explain how to install golang.
do we really need this?

File: README.md (hunk 2 above)
Comment (raproid) [reply README.md:40]: Removed

File: README.md (hunk 6 above)
Comment (raproid) [reply README.md:34]: Don't we need this since we claim users should have a specific golang version?

Comment (raproid) [reply README.md:34]: It took me a good amount of time to work out how to install a specific go version since I had 1.14.2 installed from the beginning.

Comment (raproid) [reply README.md:34]: voting to leave it be :-)

Comment (raproid) [reply README.md:34]: resolving this for now. I'll show the readme to Sonya later, ask her opinion on it and cut it if needed.

File: README.md [hunk 7]
```diff
@@ -1,45 +1,110 @@
...
+## Quick start
+
+You can test Insolar Mainnet locally:
+
+1. Install everything from the **Prerequisites** section.
+2. Install this application.
+3. Deploy this application locally.
+4. Test this application locally.
+
+**Lifehack**: you can automatically install, deploy and run functional tests via `make` with the key `functest`:
```
Comment (megge-dream) [inline README.md:30]: for install you still need to run make :)

File: README.md [hunk 8]
```diff
@@ -48,24 +115,41 @@ To test Mainnet locally, install it and deploy as described below.
      * `-k`: Path to the root user's key pair.
      * `-c`: Number of concurrent threads in which requests are sent.
      * `-r`: Number of transfer requests to be sent in each thread.
+     
+#### Functional tests
+
+These tests aim to assess operability of the app and include creating user accounts; migrating INS to Mainnet using mockups; exchanging INS for XNS using mockups; transferring XNS mockups between user accounts.
```
Comment (megge-dream) [inline README.md:121]: "transferring XNS **mockups** between user accounts" - why mockups?

File: README.md [hunk 9]
```diff
@@ -48,24 +115,41 @@ To test Mainnet locally, install it and deploy as described below.
...
+
+To run the tests, wait till the pulse watcher says `INSOLAR STATE: READY` and do the following in another terminal tab/window:
+
+1. Copy the bootstrap config from the `launchnet` folder to `launchnet/bootstrap`:
```
Comment (megge-dream) [inline README.md:125]: to `launchnet/**configs**'

File: README.md (hunk 7 above)
Comment (raproid) [reply README.md:30]: wait, so first make and then make functest?

File: README.md (hunk 9 above)
Comment (raproid) [reply README.md:125]: tnx

File: README.md (hunk 8 above)
Comment (raproid) [reply README.md:121]: wrongly thought that since we're testing, we're mocking up everything. of course XNS are real native tokens.

**Pull request merged**